install.js

```diff
@@ -8,8 +8,11 @@ var fs = require('fs')
   , os = require('os')
   , hook = path.join(__dirname, 'hook')
   , root = path.resolve(__dirname, '..', '..')
+  , spawn = require('child_process').spawnSync
```

Comment:
@nvasilakis Could you please correct it to
```suggestion
  , spawn = require('child_process').spawn
```

By https://stackoverflow.com/a/44643419/1535127 it is the fix of Travis build here and quite anticipated fix in this repo.